plugins/cohere/credentials_test.go

```diff
@@ -0,0 +1,49 @@
+package cohere
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestCredentialsProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, Credentials().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"temp file": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.AuthToken: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTk2MDYzODksImlhdCI6MTY4ODA3MDM4OSwidXNlcl9pZCI6IjdhZjBiYjFiLTU2MDQtNDI0Ny1hOWVhLWI0ZWI2ZjMzMjBiMCJ9.hCNIQcnrR371wsafGHj_xqnL9-OQbBGySN4Sfp6_wpI",
+				fieldname.URL:       "https://api.os.cohere.ai",
+				fieldname.Email:     "",
```

Comment:
Let's also add a value for the email.

Reply:
Let's use `wendy@appleseed.com` for the email as that's consistent with the usage on other shell plugins like Zendesk.